ui-admin/src/study/participants/enrolleeView/ParticipantTaskView.tsx

```diff
@@ -56,7 +56,7 @@ const ParticipantTaskView = ({ enrollee }: {enrollee: Enrollee}) => {
     <table className="table table-striped">
       <thead>
         <tr>
-          {table.getFlatHeaders().map(header => sortableTableHeader(header))}
+          {table.getFlatHeaders().map(header => tableHeader(header, { sortable: true, filterable: false }))}
```

Comment:
Configurable sorting/filtering because I didn't want to assume that we'd want filtering on every table in the UI